src/main/java/games/strategy/engine/config/client/remote/FileDownloader.java

```diff
@@ -0,0 +1,14 @@
+package games.strategy.engine.config.client.remote;
+
+import games.strategy.engine.framework.map.download.DownloadUtils;
+
+/**
+ * Humble object pattern. Wraps file downloading.
+ */
+@FunctionalInterface
+public interface FileDownloader {
+
+  DownloadUtils.FileDownloadResult download(final String url);
+
+  FileDownloader defaultDownloader = DownloadUtils::downloadToFile;
```

Comment:
Somewhat interestingly it is. Variables on interfaces are always final + static (by default).